helm/ingress-controller/values.yaml

```diff
@@ -14,17 +14,19 @@ image:
 # ingresses without ingress class annotation and ingresses of type ngrok
 ingressClass: ngrok
 
-# Agent log destination.
-log: stdout
+# This is the URL of the ngrok server to connect to. You should set this if you are using a custom ingress URL.
+serverAddr:
 # ngrok region to create tunnels in.
-region: us
+region:
 
 # The name of the K8S secret that contains the credentials for the
 # ingress controller.
-credentialsSecret:
-  name: ngrok-ingress-controller-credentials
-apiKey: ""
-authtoken: ""
+credentials:
+  secret:
+    create: false
```

Comment:
@inconshreveable we are updating how the credential is created via helm values based on some a review session we had a bit ago. I recall we discussed how it should be more explicit when its creating the secret or not. Am I remembering correctly, is this what you were getting at? Working through this a bit more, I do find it a bit cumbersome to have to specify the credentials *AND* have to say to create the secret since if I'm providing the values, it feels like a given that those should be used. Let me know. your thoughts on this, how its used can bee seen in the readme docs https://github.com/ngrok/ngrok-ingress-controller/pull/66/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R69